tests/ZfcRbacTest/Initializer/AuthorizationServiceInitializerTest.php

```diff
@@ -31,15 +33,16 @@ public function testInitializer()
         $authServiceClassName = 'ZfcRbac\Service\AuthorizationService';
         $initializer          = new AuthorizationServiceInitializer();
         $instance             = new AuthorizationAwareFake();
-        $serviceLocator       = $this->getMock('Zend\ServiceManager\ServiceLocatorInterface');
+
+        $serviceLocator = $this->prophesize(ServiceLocatorInterface::class)->willImplement(ContainerInterface::class);
         $authorizationService = $this->getMock('ZfcRbac\Service\AuthorizationService', [], [], '', false);
 
-        $serviceLocator->expects($this->once())
-            ->method('get')
-            ->with($authServiceClassName)
-            ->will($this->returnValue($authorizationService));
+        $serviceLocator
+            ->get($authServiceClassName)
+            ->willReturn($authorizationService)
+            ->shouldBeCalled();
 
-        $initializer->initialize($instance, $serviceLocator);
+        $initializer->initialize($instance, $serviceLocator->reveal());
```

Comment:
v3 service locator interface exents interop config interface, that's why I switched to prophecy here to be able to have a mock implements 2 interfaces.